week_1/HTML.md

```diff
@@ -0,0 +1,161 @@
+
+- The `head` tag contains all the metadata (data about data)
+
+- The content inside the `<title>` element will be shown in the browser's title bar or in the page's tab.
+
+- All the things you want to be rendered should be enclosed within the body tag
+
+```
+<body>
+    !Everything you want!
+</body>
+```
+
+### Some common HTML tags
+
+#### `Paragraph tag`
+The `<p>` tag is used whenever you want to render paragraphs onto the screen. Whatever text is enclosed within the tag will be treated as a paragraph.
+```
+<p> Your parapraph goes here </p>
+```
+
+#### `Heading tag`
+
+There are 6 heading tags in HTML, starting from `<h1>` to `<h6>`. Each of the tags renders the text within as a heading, with `<h1>` being the largest font-size and `<h6>` being the smallest.
+```
+<h1> Heading1 </h1>
+<h2> Heading2 </h2>
+```
+
+#### `Link tag`
+
+The link tag `<a>` is used whenever you want to link a certain piece of text to a webpage. 
+```
+<a href="URL"> Text </a>
+```
+
+#### `Image tag`
+
+The HTML `<img>` tag is used to embed an image in a web page.  The src contains the *relative* location of the image that you want to embed, and alt contains the text to be displayed in case the image couldn't be rendered.
+
+```
+<img src="iris_logo.png" alt="IRIS logo">
+```
+
+#### `Div tag`
+
+The `<div>` tag defines a division or a section in an HTML document. The tag is very commonly used as a container for HTML elements - which is then styled with CSS or manipulated with JavaScript.
+```
+<div>
+    Hey!
+</div>
+```
+
+#### `Span tag`
+
+The `<span>` tag is an inline container used to mark up a part of a text, or a part of a document. The `<span>` tag is much like the `<div>` element, but `<div>` is a block-level element and `<span>` is an inline element.
+```
+<span>
+    Hey!
+</span>
+```
+*Try out both div and span to see the difference between the two!*
+
+
+### Classes and IDs
+
+Suppose you have a number of different tags in your document, and you want to reference a set of them together, maybe for the purpose of styling, or for the purpose of adding JavaScript. This is where classes and ids come into picture.
+
+The class attribute is often used to point to a class name in a style sheet. It can also be used by a JavaScript to access and manipulate elements with the specific class name.
+
+```
+<div class="car">
+    Maruthi 800
+</div>
+
+<div class="car">
+    Maruthi Alto
+</div>
+
+<div class="bike">
+    Pulsar
+</div>
+```
+
+Here, 2 divs belong to the "car" class, and the other div belongs to the "bike" class. This is extremly helpful when you write styles and add JavaScript to your webpage, which will be discussed later on.
+
+The HTML `id` attribute is used to specify a unique id for an HTML element. The important difference between class and an id is that an id has to be unique. The functionality of class and id remains the same.
+
+```
+<div id="city">
+    Mangalore
+</div>
+```
+
+
```

Comment:
I think we can cover more here.
Introduce all form elements. 
Maybe navbar and footer (unless you explicitly want them to explore these)